jsk_perception/test/label_array_to_mask_image.test

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+  <include file="$(find jsk_perception)/sample/sample_label_array_to_mask_image.launch">
+    <arg name="gui" value="false" />
+    <arg name="queue_size" value="100" />
```

Comment:
@iory 

Is it better to set "gpu" arg to -1?
```
<arg name="gpu" value="-1"/>
```

It seems test failes becaues there is no gpu environment.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2624/checks?check_run_id=3532914923#step:5:17104

Reply:
Thank you for your comment!
https://github.com/knorth55/jsk_recognition/pull/5
I sent a PR to fix this test. 

@knorth55 Please check it.